test/outreach/read/sample_responses/sequence-using-since.json

```diff
@@ -0,0 +1,611 @@
+{
+  "rows": 5,
+  "data": [
+    {
+      "fields": {},
```

Comment:
`field` should not be empty, it should contain the values of the fields that were provided to ReadParams. 

For example, if ReadParams was:

```
connectors.ReadParams{
		ObjectName: "sequences",
		Fields:     []string{"id", "maxActivations", "createdAt"},
```
Then the output should be:
```
{
   "rows": 5,
   "data": [
      {
         "fields": {
                  "id": 1,
                  "maxActivations": 150,
                  "createdAt": "2024-07-26T06:36:16.000Z"
         }
        "raw": {
           // same as what you already have
        }
}
```